### ISSUE TITLE
Oracle DB create tables

### ISSUE THREAD

**Issue by nrmsilva** [archi-contribs/database-plugin#45]
When trying to create a connection to oracle DB, the automatic step of creating all support tables gives an error: ora-00972 identifier is too long

For some reason a specific table have more than 30 characters (Oracle max table name size).

**Comment by herve91** [comment]
Yes Oracle believes that professional database people do not need more than 30 characters. Their choice.

This said I paid attention to this limit, but I may have missed the limit for trigger names (Oracle do not have auto-increment column type either, so I need to create triggers to emulate auto-increment column).

I'll let you know ASAP.

**Comment by herve91** [comment]
I effectively had a trigger that was 34 chars long.
I also discovered an ORA-00932 when I select a CLOB in a joined request.
These two issues will be solved in the next version.

By the way, I also upgraded the driver to 18c in case you wish to use the latest database version :)

I will release the new version before the end of the week.

Best regards
Hervé

**Comment by herve91** [comment]
I just released the version 2.1.3. Please tell me if it solve all your issues.

Best regards
Hervé
